src/identity/components/userprofile/UserDefaults.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Button,
+  ButtonType,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  Form,
+} from '@influxdata/clockface'
+
+// Selectors and Context
+import {UserAccountContext} from 'src/accounts/context/userAccount'
+import {selectQuartzIdentity, selectQuartzOrgs} from 'src/identity/selectors'
+
+// Thunks
+import {updateDefaultOrgThunk} from 'src/identity/quartzOrganizations/actions/thunks'
+
+// Components
+import {DefaultAccountForm} from 'src/identity/components/userprofile/DefaultAccountForm'
+import {DefaultOrgForm} from 'src/identity/components/userprofile/DefaultOrgForm'
+
+// Constants
+import {
+  emptyAccount,
+  emptyOrg,
+} from 'src/identity/components/GlobalHeader/DefaultEntities'
+
+// Notifications
+import {notify} from 'src/shared/actions/notifications'
+import {
+  userProfileSaveError,
+  userProfileSaveSuccess,
+} from 'src/shared/copy/notifications'
+
+// Styles
+import 'src/identity/components/userprofile/UserProfile.scss'
+
+export const UserDefaults: FC = () => {
+  const dispatch = useDispatch()
+
+  const {userAccounts, handleSetDefaultAccount} = useContext(UserAccountContext)
+  const quartzOrganizations = useSelector(selectQuartzOrgs)
+
+  const identity = useSelector(selectQuartzIdentity)
+  const loggedInAccount = identity.currentIdentity.account
+
+  const accounts = userAccounts
+  const orgs = quartzOrganizations.orgs
+
+  const defaultAccount = useMemo(
+    () =>
+      accounts ? accounts.find(el => el.isDefault === true) : emptyAccount,
+    [accounts]
+  )
+  const defaultOrg = useMemo(
+    () => (orgs ? orgs.find(el => el.isDefault === true) : emptyOrg),
+    [orgs]
+  )
+
+  const [selectedAccount, setSelectedAccount] = useState(defaultAccount)
+  const [selectedOrg, setSelectedOrg] = useState(defaultOrg)
+
+  useEffect(() => {
+    setSelectedAccount(defaultAccount)
+  }, [accounts, defaultAccount])
+
+  useEffect(() => {
+    setSelectedOrg(defaultOrg)
+  }, [orgs, defaultOrg])
+
+  const userPickedNewAccount =
+    defaultAccount?.id !== selectedAccount?.id && selectedAccount !== null
+
+  const userPickedNewOrg =
+    defaultOrg?.id !== selectedOrg?.id && selectedOrg !== null
+
+  const userChangedPrefs = userPickedNewAccount || userPickedNewOrg
+
+  const saveButtonStatus =
+    userPickedNewAccount || userPickedNewOrg
+      ? ComponentStatus.Default
+      : ComponentStatus.Disabled
+
+  const handleChangeDefaults = async () => {
+    if (userChangedPrefs) {
+      try {
+        if (userPickedNewAccount) {
+          await handleSetDefaultAccount(selectedAccount.id, {
+            disablePopUps: true,
+          })
+        }
+
+        if (userPickedNewOrg) {
+          await dispatch(
+            updateDefaultOrgThunk({
+              accountId: loggedInAccount.id,
+              newDefaultOrg: selectedOrg,
+            })
+          )
+        }
+
+        dispatch(notify(userProfileSaveSuccess()))
+      } catch (err) {
+        dispatch(notify(userProfileSaveError()))
+      }
```

Comment:
With this behavior, the user will see the update didn't work - the same unsaved preference appears in the dropdown - and the 'save' button is still active.

Another behavior here could be, when the update doesn't succeed - reset the failed dropdown to the current default, which will also reset the 'save' button to inactive.